greenheart/tools/eco/hydrogen_mgmt.py

```diff
@@ -281,9 +281,11 @@ def run_h2_storage(
         h2_capacity = round(storage_hours * storage_max_fill_rate)
 
     if greenheart_config["h2_storage"]["type"] == "none":
-        h2_storage_results["h2_capacity"] = 0.0
+        greenheart_config["h2_capacity"] = 0.0
+        h2_storage_results["H2_storage_kg"] = 0.0
```

Comment:
Is the a specific reason for this to be capitalized?

Reply:
That was grandfathered in from the code before I got it. I'll change it to lower case.